index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+
+<body>
+    <main class="main">
+        <div class="buttons"> 
+            <button class="button" id="num">0</button>
+            <button class="button" id="num">1</button>
+            <button class="button" id="num">2</button>
+            <button class="button" id="num">3</button>
+            <button class="button" id="num">4</button>
+            <button class="button" id="num">5</button>
+            <button class="button" id="num">6</button>
+            <button class="button" id="num">7</button>
+            <button class="button" id="num">8</button>
+            <button class="button" id="num">9</button>
+            <button class="button" id="num">+</button>
+            <button class="button" id="num">-</button>
+            <button class="button" id="num">*</button>
+            <button class="button" id="num">/</button>
+            <button class="button" id="num">.</button>
+            <button class="button-equal" id="num">=</button>
+            <button class="button-clear" id="num">Clear</button>
```

Comment:
Multiple elements with the same `id` is not allowed. You should give each of these a unique id.